RubeesEat/wwwroot/css/site.css

```diff
@@ -0,0 +1,172 @@
+/*source: https://www.joshwcomeau.com/css/custom-css-reset/*/
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+* {
+    margin: 0;
+    font-family: "Metric Regular", Arial, Tahoma, sans-serif;
```

Comment:
Please apply the `font-family` to the `body` element. Similar to the `text-align` comment this is not the right way as it disables inheritance.